### DISCUSSION TITLE
Folding files in a commit diff

### DISCUSSION THREAD

**Discussion by marlun** [tpope/vim-fugitive#1742] [General]
Wouldn't it be nice to add folding for commit diffs? When there's multiple files it would make it easier to get an overview.

**Comment by tpope** [comment]
It's supported with `foldmethod=syntax`. This used to be the default but was removed because `foldmethod` is window local, meaning if we set it, we're also cursing future unrelated buffers with that behavior.

**Reply by marlun** [reply]
Ah, thanks! 🎉 Worked great. Let's see if I get stung by the window local "issue".
